packages/react-composites/src/composites/common/BaseComposite.tsx

```diff
@@ -110,3 +121,25 @@ const BaseContext = createContext<boolean>(false);
  * @private
  */
 const useBase = (): boolean => useContext(BaseContext);
+
+/**
+ * @private
+ * Provides a wrapper with a background color to ensure that composites always have a background color.
+ * This is necessary to ensure that composites are not transparent,
+ * and the background color of it's parent elements doesn't show through the composite.
+ */
+const WithBackgroundColor = (props: { children: React.ReactNode }): JSX.Element => {
+  const { children } = props;
+  const theme = useTheme();
+  const className = React.useMemo(
```

Comment:
`mergeStyles()` returns a string, and the computation is not costly. I would advise against memoization in this case.